recipe/meta.yaml

```diff
@@ -13,18 +13,25 @@ build:
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
It is unusual to have setuptools as a run dependency (unless there are entry point things). And also unusual to have setuptools as a build dependency without the build script reading ```python setup.py install --single-version-externally-managed --record=record.txt```.
Gradually we are starting to move towards ```pip install --no-deps .```, but that has by no means really taken hold yet.